pages/tutorials/getting_started.md.erb

```diff
@@ -12,15 +12,12 @@ This guide will help you setup your first Buildkite pipeline. If you don't have
 2. Make sure you've configured the agent token, as that is what connects the agent to your Buildkite account. Refer to [Buildkite Agent Configuration](/docs/agent/v3/configuration) for more information.
 3. Opt in to the YAML steps editor. This is the newer way of configuring pipelines. Refer to [Pipeline Upgrade - Using YAML Steps for new pipelines](/docs/tutorials/pipeline-upgrade#using-yaml-steps-for-new-pipelines) for more information.
 
-To confirm that your Buildkite agent is running, and configured correctly with your credentials, go to you organization's *Agents* page. You should see a list of all agents linked to the account, and their status.
-
-
+To confirm that your Buildkite agent is running, and configured correctly with your credentials, go to your organization's *Agents* page. You should see a list of all agents linked to the account, and their status.
```

Comment:
The main change is: _you organization's_ -> _your organization's_.

The rest is just removing the unnecessary blank lines.